### DISCUSSION TITLE
Can't run upgrade_check, even if installed

### DISCUSSION THREAD

**Discussion by cmdel** [apache/airflow#15283] [Q&A]
I'm getting a strange behaviour with trying to upgrade to v.2.0.
I've upgraded to 1.10.14 and have upgraded the db with each step. I'm running Airflow inside a docker container, based on Puckel's distribution, bumped to install 1.10.14.
I realise there are many moving parts here, but once I run up the the container:

```
> docker-compose -f docker-compose-LocalExecutor.yml run --rm webserver airflow upgrade_check
```
I get he following message:
"Please install apache-airflow-upgrade-check distribution from PyPI to perform upgrade checks"

When I connect to the container and manually install the PYPI package:
```

> docker-compose -f docker-compose-LocalExecutor.yml run --rm webserver /bin/bash
Successfully installed apache-airflow-upgrade-check-1.3.0 pandas-market-calendars-1.6.1 toolz-0.11.1 trading-calendars-2.1.1
root@cf9012fbd69d:/usr/local/airflow# airflow upgrade_check

Please install apache-airflow-upgrade-check distribution from PyPI to perform upgrade checks

```

Even though I'm installing the apache-airflow-upgrade-check package, the airflow command doesn't seem to find it.

Any ideas what I might be doing wrong?

Thanks,
Christos

**Comment by potiuk** [comment]
You need to prepare your own image to get airflow-upgrade-check installed . If you add it to a running container and start docker-compose again you start a new container which uses the base image without it. See http://airflow.apache.org/docs/apache-airflow/stable/production-deployment.html#extending-the-image on how to extend the airflow image with extra packages.
